devsetup/tripleo/tripleo_install.sh

```diff
@@ -169,6 +169,57 @@ if [ "$EDPM_COMPUTE_CEPH_ENABLED" = "true" ] ; then
     /tmp/ceph.sh
 fi
 
+if [ "$TLSE_ENABLED" = "true" ]; then
+    ENV_ARGS+=" -e /usr/share/openstack-tripleo-heat-templates/environments/ssl/tls-everywhere-endpoints-dns.yaml"
+    ENV_ARGS+=" -e /usr/share/openstack-tripleo-heat-templates/environments/services/haproxy-public-tls-certmonger.yaml"
+    ENV_ARGS+=" -e /usr/share/openstack-tripleo-heat-templates/environments/ssl/enable-internal-tls.yaml"
+    ENV_ARGS+=" -e /usr/share/openstack-tripleo-heat-templates/environments/ssl/enable-memcached-tls.yaml"
+    ENV_ARGS+=" -e /usr/share/openstack-tripleo-heat-templates/ci/environments/standalone-ipa.yaml"
+    export IPA_ADMIN_USER=admin
+    export IPA_PRINCIPAL=$IPA_ADMIN_USER
+    export IPA_ADMIN_PASSWORD=fce95318204114530f31f885c9df588f
+    export IPA_PASSWORD=$IPA_ADMIN_PASSWORD
+    export UNDERCLOUD_FQDN=undercloud.$CLOUD_DOMAIN
+    export IPA_DOMAIN=$CLOUD_DOMAIN
+    export IPA_REALM=$(echo $IPA_DOMAIN | awk '{print toupper($0)}')
+    export IPA_HOST=ipa.$IPA_DOMAIN
+    export IPA_SERVER_HOSTNAME=$IPA_HOST
+    sudo mkdir /tmp/ipa-data
+    sudo podman run -d --name freeipa-server-container \
+        --sysctl net.ipv6.conf.lo.disable_ipv6=0 \
+        --security-opt seccomp=unconfined \
+        --ip 10.255.255.25 \
+        -e IPA_SERVER_IP=10.255.255.25 \
+        -e PASSWORD=$IPA_ADMIN_PASSWORD \
+        -h $IPA_SERVER_HOSTNAME \
+        --read-only --tmpfs /run --tmpfs /tmp \
+        -v /sys/fs/cgroup:/sys/fs/cgroup:ro \
+        -v /tmp/ipa-data:/data:Z quay.io/freeipa/freeipa-server:fedora-39 no-exit \
+        -U -r $IPA_REALM --setup-dns --no-reverse --no-ntp \
+        --no-dnssec-validation --auto-forwarders
+    timeout 900s grep -qEi '(INFO The ipa-server-install command was successful|ERROR The ipa-server-install command failed)' <(sudo tail -F /tmp/ipa-data/var/log/ipaserver-install.log)
+    cat  <<EOF > ipa_resolv.conf
```

Comment:
need this on all the overcloud nodes too

Reply:
@marios I'm thinking that it might be a good idea to leave a note here, that this code assumes that the overcloud nodes are setup to have the ipa server as dns resolver. We do that in CI from a rdo-jobs playbook, but it's not obvious from inspection of the code here

Reply:
ack